### DISCUSSION TITLE
How to use custom css files?

### DISCUSSION THREAD

**Discussion by raj-pulapakura** [chakra-ui/chakra-ui#5249] [Q&A]
Hi everyone,  I was wondering how to use my own custom css files along with chakra ui. It doesn't seem to work - I don't know if this is intended or if it is a bug. Thanks!

**Comment by primos63** [comment]
I understand what you wrote, but I have no idea what the problem is from just that. What's not working? Why do you think it's not working? The best way to get your question answered is to provide some sort of example showing the problem either through CodeSandbox or one of the other online tools like it.

**Reply by raj-pulapakura** [reply]
Never mind, I switched to Material UI. Thanks for your reply anyways.
